### PR TITLE
Capitalized GitHub in comment replies, basic a-centralized-mirror support

### PR DESCRIPTION
**Pull request by kyleratti**
It's `GitHub` not `Github` - critical bug that needs fixing.

I've also included _super_ basic support for a-centralized-mirror. A few things to mention:

- I haven't written Python in 5 years now, and I'm pretty sure the last time I did it was Python 2. This appears to be Python 3.
- **This is untested**, but VS Code doesn't show any syntax or usage errors
- I included _absolutely **no**_ error handling in this
- I would probably separate the reddit reply and acm updates into their own functions with their own error handling

### CODE REVIEW

**Comment by wferrell545** [comment]
You have a typo in the header method. The letter "R" is capitalized. Which is iRonic because you claim the "H" in "Github" is not capitalized.

I'm done.

**Comment by wferrell545** [comment]
But I'm not done with this pr.

Why do you see fit to deliver this code without any test to prove it is indeed doing what you claim it does?

File: PublicFreakout.py [hunk 1]
```diff
@@ -221,10 +226,26 @@ def reply_reddit(submission, mirror_url):
...
+                "data": {
+                    "redditPostId": submission.permalink,
+                    "mirrorUrl": "http://mirrorbot.ga" + submission.permalink
+                }
+            }
+            post_data_encoded = urllib.parse.urlencode(post_data)
+            post_request = urllib.request.Request(post_url, data=post_data)
+            post_request.add_header("X-ACM-API-Token", acm_api_token)
+            post_request.add_headeR("X-ACM-Bot-Token", acm_bot_token)
+            post_response = post_request.urlopen(post_request)
```
Comment (wferrell545) [inline PublicFreakout.py:248]: Typo

**Comment by kyleratti** [comment]
@wferrell545

> You have a typo in the header method. The letter "R" is capitalized. Which is iRonic because you claim the "H" in "Github" is not capitalized.

It's not a claim - it isn't capitalized!

> Why do you see fit to deliver this code without any test to prove it is indeed doing what you claim it does?

1. As mentioned, I haven't written Python in a very long time. Unfortunately I no longer possess the knowledge of best-practices when writing it and I'm not going to re-learn it for a sample implementation on a cousin mirror-bot project.
2. It's a proof of concept and starting point as I've discussed this with @jonnotjohn multiple times over the past 6 months or so while the project was in planning and development
3. I totally welcome your Python knowledge and therefore any/all modifications and tests you'd like to provide on this pull request. As I've said, it's not my wheelhouse anymore so I would love the input of someone more qualified than me!
4. The published code in this repo is not current  so it will need to be manually adapted into the newer code base regardless of whether or not I implemented error handling

**Comment by kyleratti** [comment]
Yes, it's ironic that the 'r' was capitalized. Not sure why VSCode didn't catch that but a pull request could've fixed that without the attitude. I don't know if it's the tone given off because it's written text and not spoken or if it's intentional but I don't see the reason or need for the attitude.

**Comment by wferrell545** [comment]
My point is if you had tested this then you would have caught the typo. Which tells me this is untested code. I know that you've said that you haven't tested this. That's not what I am arguing.  Are we ok with the fact we have untested code? 

I'm not.

**Comment by kyleratti** [comment]
> My point is if you had tested this then you would have caught the typo. Which tells me this is untested code. I know that you've said that you haven't tested this. That's not what I am arguing. Are we ok with the fact we have untested code?
> 
> I'm not.

I await the results of your testing, though they will be invalid since the proposed code change is not against the current source of the project.

It's a reddit bot side project, not a giant corporate application with full change control procedures - relax.

**Review by wferrell545 — APPROVED** [review]

**Pull request closed without merging**